tests/arrow_tests.py

```diff
@@ -1479,9 +1485,63 @@ def test_granularity(self):
             ),
             "3 years",
         )
+
         with self.assertRaises(AttributeError):
             self.now.humanize(later108, granularity="years")
 
+    def test_multiple_granularity(self):
+        self.assertEqual(self.now.humanize(granularity="second"), "just now")
+        self.assertEqual(self.now.humanize(granularity=["second"]), "just now")
+        self.assertEqual(
+            self.now.humanize(granularity=["year", "month", "day", "hour", "second"]),
+            "in 0 years 0 months 0 days 0 hours and seconds",
+        )
+
+        later4000 = self.now.shift(seconds=4000)
+        self.assertEqual(
+            later4000.humanize(self.now, granularity=["hour", "minute"]),
+            "in an hour and 6 minutes",
+        )
+        self.assertEqual(
+            self.now.humanize(later4000, granularity=["hour", "minute"]),
+            "an hour and 6 minutes ago",
+        )
+        self.assertEqual(
+            later4000.humanize(
+                self.now, granularity=["hour", "minute"], only_distance=True
+            ),
+            "an hour and 6 minutes",
+        )
+        self.assertEqual(
+            later4000.humanize(self.now, granularity=["day", "hour", "minute"]),
+            "in 0 days an hour and 6 minutes",
+        )
+        self.assertEqual(
+            self.now.humanize(later4000, granularity=["day", "hour", "minute"]),
+            "0 days an hour and 6 minutes ago",
+        )
+
+        later105 = self.now.shift(seconds=10 ** 5)
+        self.assertEqual(
+            self.now.humanize(later105, granularity=["hour", "day", "minute"]),
+            "a day 3 hours and 46 minutes ago",
+        )
+        with self.assertRaises(AttributeError):
+            self.now.humanize(later105, granularity=["error", "second"])
+
+        later108onlydistance = self.now.shift(seconds=10 ** 8)
+        self.assertEqual(
+            self.now.humanize(later108onlydistance, granularity=["year"]), "3 years ago"
+        )
+        self.assertEqual(
+            self.now.humanize(later108onlydistance, granularity=["month", "week"]),
+            "37 months and 4 weeks ago",
+        )
+        self.assertEqual(
+            self.now.humanize(later108onlydistance, granularity=["year", "second"]),
+            "3 years and seconds ago",
+        )
```

Comment:
You forgot to set `only_distance=False` in the `humanize` function. With `only_distance` set to true, the first one should be `3 years` since `only_distance` removes the `ago` and `in` (ref: https://arrow.readthedocs.io/en/latest/#arrow.arrow.Arrow.humanize)

Reply:
seems like I forgot to pass only_distance=True into the parameters for humanize in this section so the variable looks mis-named and the output looks wrong. In newest commit I passed in this parameter and updated the asserted output to not have any "ago" or "in" words. The humanize functionality is behaving correctly though